<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="6" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-07-01T20:21:15 IST" name="Default suite" finished-at="2023-07-01T20:21:31 IST" duration-ms="15753">
    <groups>
    </groups>
    <test started-at="2023-07-01T20:21:15 IST" name="Default test" finished-at="2023-07-01T20:21:31 IST" duration-ms="15753">
      <class name="test_classes.patch_TC1">
        <test-method signature="extractor()[pri:0, instance:test_classes.patch_TC1@1040be71]" started-at="2023-07-01T20:21:26 IST" name="extractor" finished-at="2023-07-01T20:21:27 IST" duration-ms="1714" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractor -->
      </class> <!-- test_classes.patch_TC1 -->
      <class name="test_classes.Put_Retry_TC1">
        <test-method signature="execute()[pri:0, instance:test_classes.Put_Retry_TC1@6f10d5b6]" started-at="2023-07-01T20:21:24 IST" name="execute" finished-at="2023-07-01T20:21:26 IST" duration-ms="1560" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- execute -->
      </class> <!-- test_classes.Put_Retry_TC1 -->
      <class name="test_classes.post_TC1">
        <test-method signature="extractor()[pri:2, instance:test_classes.post_TC1@548a24a]" started-at="2023-07-01T20:21:27 IST" name="extractor" finished-at="2023-07-01T20:21:29 IST" duration-ms="1562" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractor -->
      </class> <!-- test_classes.post_TC1 -->
      <class name="test_classes.Post_Retry_TC1">
        <test-method signature="validator(java.lang.String,java.lang.String)[pri:1, instance:test_classes.Post_Retry_TC1@6d60fe40]" started-at="2023-07-01T20:21:15 IST" name="validator" finished-at="2023-07-01T20:21:15 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Cannot inject @Test annotated Method [validator] with [class java.lang.String, class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [validator] with [class java.lang.String, class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:436)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:350)
at org.testng.internal.Parameters.createParameters(Parameters.java:693)
at org.testng.internal.Parameters.handleParameters(Parameters.java:879)
at org.testng.internal.Parameters.handleParameters(Parameters.java:728)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:911)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validator -->
        <test-method signature="execute()[pri:2, instance:test_classes.Post_Retry_TC1@6d60fe40]" started-at="2023-07-01T20:21:15 IST" name="execute" finished-at="2023-07-01T20:21:19 IST" duration-ms="4495" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- execute -->
      </class> <!-- test_classes.Post_Retry_TC1 -->
      <class name="test_classes.put_TC1">
        <test-method signature="extractor()[pri:0, instance:test_classes.put_TC1@433defed]" started-at="2023-07-01T20:21:29 IST" name="extractor" finished-at="2023-07-01T20:21:31 IST" duration-ms="1599" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractor -->
      </class> <!-- test_classes.put_TC1 -->
      <class name="test_classes.Patch_Retry_TC1">
        <test-method signature="execute()[pri:0, instance:test_classes.Patch_Retry_TC1@61c4eee0]" started-at="2023-07-01T20:21:19 IST" name="execute" finished-at="2023-07-01T20:21:24 IST" duration-ms="4635" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- execute -->
      </class> <!-- test_classes.Patch_Retry_TC1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
